cli/src/utils/loadLocalCommands.ts

```diff
@@ -0,0 +1,142 @@
+import fs from "fs-extra";
+import path from "path";
+import * as glob from "glob";
+
+export async function loadLocalCommands(program) {
+  // are we in a grouparoo project directory?
+  const localPackageFile = path.join(process.cwd(), "package.json");
+  if (!fs.existsSync(localPackageFile)) return;
+  const pkgJSON = readPackageJSON(localPackageFile);
+  if (!Object.keys(pkgJSON?.dependencies || {}).includes("@grouparoo/core")) {
+    return;
+  }
+
+  try {
+    const { config } = getActionhero();
+
+    let pathsLoaded: string[] = [];
+
+    // grouparoo core
+    for (const i in config.general.paths.cli) {
+      await loadDirectory(
+        program,
+        path.join(config.general.paths.cli[i]),
+        pathsLoaded
+      );
+    }
+
+    // plugins
+    for (const pluginName in config.plugins) {
+      if (config.plugins[pluginName].cli !== false) {
+        await loadDirectory(
+          program,
+          path.join(config.plugins[pluginName].path, "dist", "bin"),
+          pathsLoaded
+        );
+      }
+    }
+  } catch (error) {
+    process.chdir(process.env.INIT_CWD);
+    throw error;
+  }
+}
+
+// --- UTILS ---
+
+function getActionhero() {
+  process.chdir(
```

Comment:
what's the story here? should we put this back?
could we require without changing dirs?

Reply:
We change back to the original CWD here https://github.com/grouparoo/grouparoo/pull/1070/files#diff-e42b162691ea4dfc92fcb12a614b7cf16041729a65b645373f1520ea8d85323aR40-R42

And no... There's a lot of shenanigans we are doing about running the main app form /within/ `node_modules`, that we really need to always run everything from that working directory (it's why we `cd` in the package.json commands too).  It's further complicated because actionhero is a child of grouparoo is a child of the root project...

Reply:
Oh, and we want to *stay* in the changed working directory if we are running any command that needs to start up the application